include/ignition/launch/Plugin.hh

```diff
@@ -30,6 +30,9 @@ namespace ignition
     /// \brief Base class for launch plugins.
     class Plugin
     {
+      // Default destructor
+      public: virtual ~Plugin() {}
```

Comment:
ABI checker says that this breaks ABI. Does that matter?

Reply:
I think it's safe to ignore that. If a class inherited from `Plugin` and implemented a destructor, then this will allow that destructor to be called correctly. And it won't break their code. In a similar vein, if a child did not implement a destructor then the code should also be safe.

Reply:
the size of the v-table is changed, so if there are any libraries that link against gz-launch plugins, this ABI change would break them. There probably aren't many things linking against these plugins though

Reply:
I think we can choose to ignore it but wanted to be explicit about it